File: dotnet/fileLoader.cs
using System;
using System.IO;
using System.Linq;

class fileLoader
{
	static public void Main(String[] args)
	{		
		new fileLoader().Cli(args);
	}
	
	public void Cli(String[] args)
	{
		if (args.Length == 0)
		{
			Console.WriteLine("Podaj nazwe pliku byku");
		}
		
		else if (args.Length > 1)
		{
			Console.WriteLine("Podaj tylko jeden plik");
		}
		
		else if (File.Exists(args[0]))
		{
			string text = File.ReadAllText(args[0]);			
			string reversedText = this.Reverse(text);
			
			File.WriteAllText(args[0], reversedText);
			Console.WriteLine("Poprawnie odwrócono treść pliku");
		}
		
		else
		{
			Console.WriteLine("Nie ma takiego pliku wariacie");
		}
	}
	
	public string Reverse (string text)
	{		
		string reversed = string.Join(" ", text.Split(' ').Select(s => new String(s.Reverse().ToArray())));
		
		return reversed;		
	}
}

File: rust/fileLoader.rs
use std::path::Path;
use std::fs;
use std::env;

fn main()
{
	let args: Vec<_> = env::args().collect();
	
	if args.len() == 1
	{
		print!("Podaj nazwe pliku byku");
	}
	
	else if args.len() > 2
	{
		print!("Podaj tylko jeden plik");
	}
	
	else if Path::new(&args[1]).exists()
	{
		let text = fs::read_to_string(&args[1]).expect("Unable to read file");
		let words_iterator = text.split(" ");
		
		let words_vector: Vec<&str> = words_iterator.collect();
		words_vector.join(" ");
		
		
		fs::write(&args[1], &text.chars().rev().collect::<String>()).expect("Unable to write file");
		
		print!("Poprawnie odwrocono tresc pliku");
	}
	else
	{
		print!("Nie ma takiego pliku wariacie");
	}
}

